packages/grafana-data/src/themes/palette.ts

```diff
@@ -16,7 +16,7 @@ export const palette = {
   darkBorder2: '#64646B',
 
   // Dashboard bg / layer 0 (light theme)
-  gray90: '#F4F5F5',
+  gray90: '#FFFFFF',
```

Comment:
Instead of changing the meaning of these, can we add a new theme that's got our constants involved?  That way we don't have merge hell in the future.

Reply:
What do you think now?